roles/bootstrap-os/tasks/bootstrap-oracle.yml

```diff
@@ -0,0 +1,84 @@
+---
+
+- name: Check presence of fastestmirror.conf
+  stat:
+    path: /etc/yum/pluginconf.d/fastestmirror.conf
+  register: fastestmirror
+
+# fastestmirror plugin actually slows down Ansible deployments
+- name: Disable fastestmirror plugin
```

Comment:
Please put this behind a (documented) variable, e.g. like for CentOS/RHEL: https://github.com/kubernetes-sigs/kubespray/tree/master/roles/bootstrap-os#centosrhel

Reply:
👍